### PR TITLE
Issue668: update file drop and menu if drop location changes

### PR DESCRIPTION
**Pull request by Jeff-Wang93**
#668

### CODE REVIEW

File: vent/menus/main.py [hunk 1]
```diff
@@ -134,6 +136,24 @@ def while_waiting(self):
         self.addfield6.display()
         self.addfield7.display()
 
+        # if file drop location changes deal with it
+        logger = Logger(__name__)
+        if self.file_drop.value != DropLocation():
```
Comment (cglewis) [inline vent/menus/main.py]: this can be reused for the initial value of `self.file_drop` correct, rather than doing that check in `create`?

Comment (cglewis) [inline vent/menus/main.py]: around line 390

Comment (Jeff-Wang93) [inline vent/menus/main.py]: I'm afraid I don't quite understand what the question is. I'm not doing a check inside `create`.

File: vent/menus/main.py [hunk 2]
```diff
@@ -387,8 +387,6 @@ def create(self):
         try:
             self.api_action = Action()
             drop_location = os.path.join(PathDirs().base_dir, "vent.cfg")
```
Comment (cglewis) [inline vent/menus/main.py]: i don't think this line is necessary anymore?

Comment (Jeff-Wang93) [inline vent/menus/main.py]: ah right. I'll get rid of that

Comment (cglewis) [inline vent/menus/main.py]: also the `Template` import

**Pull request merged**